### ISSUE TITLE
end-of-file line breaks - three, one could be enough (end-of-file-fixer pre-commit)

### ISSUE THREAD

**Issue by arthurzenika** [vemel/handsdown#27]
**Describe the bug**
We use end-of-file-fixer pre-commit on our project https://pre-commit.com/hooks.html which checks that files have one line break at the end of versioned files. On markdown generated by handsdown it removes two line breaks. 

One can ignore generated files with :

```
repos:
-   repo: https://github.com/pre-commit/pre-commit-hooks
    rev: v4.4.0
    hooks:
    -   id: end-of-file-fixer
        exclude: ^docs/module/.*$
```

**To Reproduce**
1. Configure pre-commit with end-of-file-fixer (see config above)
2. Run `pre-commit run --all-files`
3. Result : `git diff` 

![Screenshot from 2023-01-10 14-28-40](https://user-images.githubusercontent.com/445200/211564394-8e4218af-4822-4d7c-b40e-265a413c13d6.png)

**Expected behavior**
Maybe it would be nice if handsdown generated less line breaks

**Comment by vemel** [comment]
Hello! Thank you for the report.

Yes, I tried to reduce the number of line breaks. I will try to make it compatible with `end-of-file-fixer`

**Comment by vemel** [comment]
I have just released version `2.0.1` with the fix included. Please try it out and let me know if it works for you.
